core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.component.ts

```diff
@@ -204,6 +204,14 @@ export class SkillConceptCardEditorComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.directiveSubscriptions.add(
+      this.windowDimensionsService.getResizeEvent().subscribe(
```

Comment:
Add a test for an emit event.
Ditto for all other places.

Reply:
PTAL to the changes! now all tests are passing.
thanks!

Reply:
event emit tests are already there. Please confirm @gp201 
thanks!

Reply:
I assume you're referring to this event emit test:
https://github.com/oppia/oppia/blob/bad04d731dafbc7e31a64f1feafd5dc45f3fdcfc/core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.component.spec.ts#L115-L123

I believe this test only tests this subscription:
https://github.com/oppia/oppia/blob/bad04d731dafbc7e31a64f1feafd5dc45f3fdcfc/core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.component.ts#L221-L228

You can refer to [activity-tiles-infinity-grid.component.spec.ts](https://github.com/oppia/oppia/blob/14cebfad0fd3e21c342ea79ab034e31842b1255a/core/templates/pages/library-page/search-results/activity-tiles-infinity-grid.component.spec.ts) file to see how to test it.